prisma/prisma.yml

```diff
@@ -1,25 +1,24 @@
 # this can be an environment variabel OR the endpoint from the initial prisma init/deploy
-endpoint:  "http://localhost:4466"
+endpoint: 'http://localhost:4466'
 
-datamodel: 
+datamodel:
   - types.graphql
   - enums.graphql
 
 generate:
   - generator: typescript-client
     output: ../src/generated/prisma-client/
- 
+  - generator: graphql-schema
```

Comment:
we also have a schema generator so you don't need graphql get-schema!

Reply:
great :-) . I did not realise that. It is not in your tutorials yet as far as I can see

Reply:
... sorry about all these comment. How do I run the prisma schema generator?

Reply:
even more sorry. I guess it s the added generator part in prisma.yml :-)